#161278903 Integrate Coveralls code coverage service to application

susanwere opened the pull request:
[![Coverage Status](https://coveralls.io/repos/github/andela/ah-sealteam/badge.svg?branch=ch-intergrate-coveralls-161278903)](https://coveralls.io/github/andela/ah-sealteam?branch=ch-intergrate-coveralls-161278903) [![Build Status](https://travis-ci.org/andela/ah-sealteam.svg?branch=ch-intergrate-coveralls-161278903)](https://travis-ci.org/andela/ah-sealteam)

**What does this PR do?**

It integrates coveralls code coverage service with readme badge to the application

**Description of Task to be completed?**

1. Add repo to coveralls website
2. Install coveralls in application
3. Create .coveralls.yml file and add the neccessary dependencies 

              - add service name
              - add repo token
 
4. Update travis.yml file

           - update travis file to run tests with coverage
           - add command to run coveralls after_success

**How should this be manually tested?**

Click or view the added badge in order to view the build status of the application

**What are the relevant pivotal tracker stories?**

[#161278903](https://www.pivotaltracker.com/story/show/161278903)

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

rickynyairo reviewed (approved):
Add the .vscode folder to the .gitignore file since the settings under that settings.json file are custom to your workspace.

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

Commit:
chore(tests): integrate coveralls code coverage service
- add repo to coveralls website
- install coveralls in application
- update travis file to run tests with coverage
- add command to run coveralls after_success
[Finishes #161278903]

susanwere committed:
Update README.md

proxiex committed:
Integrate coveralls
[Chore #161278892] Add set up for pytest test runner

susanwere committed:
Merge branch 'develop' into ch-intergrate-coveralls-161278903

proxiex commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
```
Add this file to git ignore.

susanwere replied:
Okay. Done

susanwere committed:
Update README.md

susanwere committed:
[Chore #161278903] Intergrate coveralls and travis

susanwere committed:
[Chore #161278903] Intergrate coveralls and travis

susanwere committed:
[Chore #161278903] Intergrate coveralls and travis

susanwere committed:
[Chore #161278903] Intergrate coveralls and travis

susanwere committed:
chore(tests): intergrate coveralls [Finishes #161278903]

susanwere committed:
chore(tests): intergrate coveralls [Finishes #161278903]

susanwere committed:
Merge branch 'ch-intergrate-coveralls-161278903' of https://github.com/andela/ah-sealteam into ch-intergrate-coveralls-161278903

susanwere committed:
chore(tests): intergrate coveralls  [Finishes #161278903]

susanwere committed:
chore(tests): intergrate coveralls  [Finishes #161278903]
